Assets/Scripts/Score/ScoreManager.cs

```diff
@@ -0,0 +1,71 @@
+using UnityEngine;
+using UnityEngine.SceneManagement;
+using TMPro;
+
+public class ScoreManager : MonoBehaviour
+{
+    public static ScoreManager Instance { get; private set; }
+    public float Score
+    {
+        get { return score; }
+        private set { score = value; }
+    }
+
+    [SerializeField]
+    private TextMeshProUGUI scoreText;
+
+    private const float moveMulti4Pts = 10.0f, // The value to multiply the distance traveled by
+                        enemyFelledPts = 100.0f; // The points killing an enemy is worth
+
+    private float score,
+                  displayScore,
+                  pastScore,
+                  startY,
```

Comment:
`startY` -> `startPlayerYPos` / `startPlayerYPosition` (or something like that).